jujugui/static/gui/src/app/utils/github-ssh-keys.js

```diff
@@ -37,7 +37,13 @@ if (typeof this.jujugui.sshKeys === 'undefined') {
 const githubSSHKeys = (handler, username, callback) => {
   // On load, call the callback with munged data.
   const wrap = (response) => {
-    let data = JSON.parse(response.currentTarget.response);
+    let data = null;
+    try {
+      data = JSON.parse(response.currentTarget.response);
+    } catch(e) {
+      callback('Could not parse response', null);
```

Comment:
Include the response text in the error so that it is easier to debug?
Also maybe: 'cannot retrieve SSH keys from gihub: invalid response: ....' ?